### PR TITLE
ChillAndUnbond for Polkadot

### PR DESCRIPTION
**Pull request by rsrbk**
## Description

As per mobile UX we want to be able to unbond any transaction from a single screen, which, in case of unbonding 100% of the staked amount, would require to send `Chill` transaction first.

## Types of changes

* New feature (non-breaking change which adds functionality) 

## Checklist

- [x] Create pull request as draft initially, unless its complete.
- [x] Add tests to cover changes as needed.
- [ ] Update documentation as needed.
- [ ] If there is a related Issue, mention it in the description.

If you're adding a new blockchain

- [ ] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain.

### CODE REVIEW

File: src/Polkadot/Extrinsic.cpp [hunk 1]
```diff
@@ -25,6 +25,7 @@ static const std::string stakingUnbond = "Staking.unbond";
 static const std::string stakingWithdrawUnbond = "Staking.withdraw_unbonded";
 static const std::string stakingNominate = "Staking.nominate";
 static const std::string stakingChill = "Staking.chill";
+static const std::string stakingChillAndUnbond = "Staking.chill_and_unbond";
```
Comment (hewigovens) [inline src/Polkadot/Extrinsic.cpp:28]: There is no `chill_and_unbond` in polkadot runtime, no need to add this one

File: src/Polkadot/Extrinsic.cpp [hunk 2]
```diff
@@ -35,6 +36,7 @@ static std::map<const std::string, Data> polkadotCallIndices = {
     {stakingWithdrawUnbond, Data{0x07, 0x03}},
     {stakingNominate,       Data{0x07, 0x05}},
     {stakingChill,          Data{0x07, 0x06}},
+    {stakingChillAndUnbond, Data{0x07, 0x07}},
```
Comment (hewigovens) [inline src/Polkadot/Extrinsic.cpp:39]: no need to add call index

File: src/Polkadot/Extrinsic.cpp [hunk 3]
```diff
@@ -46,6 +48,7 @@ static std::map<const std::string, Data> kusamaCallIndices = {
     {stakingWithdrawUnbond, Data{0x06, 0x03}},
     {stakingNominate,       Data{0x06, 0x05}},
     {stakingChill,          Data{0x06, 0x06}},
+    {stakingChillAndUnbond, Data{0x06, 0x07}},
```
Comment (hewigovens) [inline src/Polkadot/Extrinsic.cpp:51]: same here

File: src/Polkadot/Extrinsic.cpp [hunk 4]
```diff
@@ -208,6 +211,32 @@ Data Extrinsic::encodeStakingCall(const Proto::Staking& staking, TWSS58AddressTy
             append(data, getCallIndex(network, stakingChill));
             break;
 
+        case Proto::Staking::kChillAndUnbond:
```
Comment (hewigovens) [inline src/Polkadot/Extrinsic.cpp:214]: Have you sent this tx to network test?

Comment (rsrbk) [reply src/Polkadot/Extrinsic.cpp:214]: not yet, figuring out how to run the console to test it

Comment (hewigovens) [reply src/Polkadot/Extrinsic.cpp:211]: wallet console doesn't have this feature, currently we have to manually send it

**Review by hewigovens — APPROVED** [review]

**Pull request merged**